README.md

```diff
@@ -2,13 +2,13 @@
 
 ## Release Notes
 
-- The current implementation of the default storage class along with its usage of an NFS server as its
-backing store may cause problems with the SAS Viya Software and its Information Catalog component. This
-issue is known and is being addressed with [this issue](https://github.com/sassoftware/viya4-iac-k8s/issues/6). As soon as this is resolved this banner will be removed and documentation will be updated to direct you on how to active the correct storage class needed.
+- The current implementation of the default storage class, along with its usage of an NFS server as its
+backing store, might cause problems with the SAS Viya software and its information catalogue component. This
+known problem is being addressed with [this issue](https://github.com/sassoftware/viya4-iac-k8s/issues/6). As soon as this problem is resolved, this banner will be removed and documentation will be updated to direct you in activating the required storage class.
```

Comment:
An issue with the implementation of the default storage class and its usage of an NFS server as its
backing store has been addressed with [this issue](https://github.com/sassoftware/viya4-iac-k8s/issues/6).